gast_to_code/gast_to_code_main.py

```diff
@@ -15,6 +15,18 @@ def list_helper(gast_list, out_lang, csv_delimiter = ", "):
     
     return out
 
+def gast_to_py_dict(gast):
+    return "{" + gast_to_code(gast["elts"], "py") + "}"
```

Comment:
How neatly the recursion is working here makes me happy :).

Reply:
It really makes implementing new features much easier!